packages/ui5-middleware-livereload/lib/livereload.js

```diff
@@ -46,7 +46,10 @@ module.exports = async ({ resources, options }) => {
         watchPath = options.configuration.watchPath || options.configuration.path;
     }
     let exclusions = [];
-    const aOptExclusions = options.configuration.exclusions;
+    let aOptExclusions;
+    if (options.configuration && options.configuration.exclusions) {
+        aOptExclusions = options.configuration.exclusions;
+    }
     if (options.configuration && aOptExclusions && Array.isArray(aOptExclusions)) {
```

Comment:
you could have removed the options.configurations check then in the aOptExclusions check - but anyway - this middleware may need a bit of refactoring soon. I will release it like this for now.